chans/sqlc/sql.go

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright 2021 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Package sqlc provides a Plax channel type for talking to a SQL
+// database.
+package sqlc
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	"plugin"
+
+	"github.com/Comcast/plax/dsl"
+
+	_ "modernc.org/sqlite"
```

Comment:
Should we use plugins?

Reply:
I thought it might be best to have something working out of the box. This `modernc.org/sqlite` is pretty cool. It's a `cgo`-free SQLite clone that seems to pass a lot of SQLite tests. Worth the trouble?

I also have `demos/mysql.yaml` which demonstrates how to use a Go plug-in to provide a driver at runtime.